From d302f7e8e9cc4c83b5af0da3ffbb3feb579e594f Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Wed, 22 Jun 2016 11:16:50 +0800
Subject: [PATCH] UPSTREAM: ARM: dts: rockchip: rename rk3228.dtsi to
 rk322x.dtsi

We have the brother chipset that RK3228 and RK3229, they share most
of dts configuration, but there are a number of different features.
In order to develop the future when they are easy to distinguish,
we need them to be independent.

Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
(cherry picked from git://git.kernel.org/pub/scm/linux/kernel/git/mmind/linux-rockchip.git
v4.8-armsoc/dts32 commit 8372d93df7c265a67676dcd329da9588cc769192)

Change-Id: I0b3c91cddb3ca919b165ba1ec5b2b9466945546d
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm/boot/dts/rk3228-evb.dts               | 2 +-
 arch/arm/boot/dts/{rk3228.dtsi => rk322x.dtsi} | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)
 rename arch/arm/boot/dts/{rk3228.dtsi => rk322x.dtsi} (99%)

diff --git a/arch/arm/boot/dts/rk3228-evb.dts b/arch/arm/boot/dts/rk3228-evb.dts
index 5956e8246abe..904668e2e666 100644
--- a/arch/arm/boot/dts/rk3228-evb.dts
+++ b/arch/arm/boot/dts/rk3228-evb.dts
@@ -40,7 +40,7 @@
 
 /dts-v1/;
 
-#include "rk3228.dtsi"
+#include "rk322x.dtsi"
 
 / {
 	model = "Rockchip RK3228 Evaluation board";
diff --git a/arch/arm/boot/dts/rk3228.dtsi b/arch/arm/boot/dts/rk322x.dtsi
similarity index 99%
rename from arch/arm/boot/dts/rk3228.dtsi
rename to arch/arm/boot/dts/rk322x.dtsi
index 7779896bfee6..0fec9b860aa1 100644
--- a/arch/arm/boot/dts/rk3228.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -47,8 +47,6 @@
 #include "skeleton.dtsi"
 
 / {
-	compatible = "rockchip,rk3228";
-
 	interrupt-parent = <&gic>;
 
 	aliases {
-- 
2.35.3

